server/data/personIntegrationApiClient.ts

```diff
@@ -14,10 +14,18 @@ export default class PersonIntegrationApiRestClient implements PersonIntegration
   }
 
   updateBirthPlace(prisonerNumber: string, birthPlace: string): Promise<void> {
+    return this.updateCorePersonRecord(prisonerNumber, 'BIRTHPLACE', birthPlace)
+  }
+
+  updateNationality(prisonerNumber: string, nationality: string): Promise<void> {
+    return this.updateCorePersonRecord(prisonerNumber, 'NATIONALITY', nationality)
+  }
+
+  private updateCorePersonRecord(prisonerNumber: string, fieldName: string, value: string): Promise<void> {
```

Comment:
Nice spot that we can reuse this. We can apply this to `updateCountryOfBirth` too